src/shogun/evaluation/CrossValidationStorage.h

```diff
@@ -155,7 +155,7 @@ namespace shogun
 		std::shared_ptr<Labels> m_original_labels;
 
 		/** Vector with all the folds results */
-		std::vector<std::shared_ptr<CrossValidationFoldStorage>> m_folds_results;
+		std::vector<std::shared_ptr<EvaluationResult>> m_folds_results;
```

Comment:
@karlnapf is there a reason why this was CrossValidationFoldStorage instead of the base class EvaluationResult?